frontend/src/scenes/billing/BillingSubscribed.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button, Col, Row } from 'antd'
+import React from 'react'
+import { WelcomeLogo } from 'scenes/authentication/WelcomeLogo'
+import './BillingSubscribed.scss'
+import hedgehogMain from 'public/hedgehog-bridge-page.png'
+import { HelpButton } from 'lib/components/HelpButton/HelpButton'
+import { CheckCircleOutlined, CloseCircleOutlined, ArrowRightOutlined } from '@ant-design/icons'
+import { useActions, useValues } from 'kea'
+import { router } from 'kea-router'
+import { Link } from 'lib/components/Link'
+import { billingSubscribedLogic, SubscriptionStatus } from './billingSubscribedLogic'
+
+export function BillingSubscribed(): JSX.Element {
+    const { status } = useValues(billingSubscribedLogic)
+
+    return (
+        <div className="bridge-page billing-subscribed">
+            <Row>
+                <Col span={24} className="auth-main-content">
+                    <img src={hedgehogMain} alt="" className="main-art" />
+                    <div className="inner-wrapper">
+                        <WelcomeLogo view="signup" />
+                        <div className="inner">
+                            {status === SubscriptionStatus.Success ? <SubscriptionSuccess /> : <SubscriptionFailure />}
+                            <div className="support-footer">
+                                Have questions? <HelpButton customComponent={<a href="#">Get help</a>} />
+                            </div>
+                        </div>
+                    </div>
+                </Col>
+            </Row>
+        </div>
+    )
+}
+
+function SubscriptionSuccess(): JSX.Element {
+    const { push } = useActions(router)
+    const { billing } = useValues(billingSubscribedLogic)
+
+    return (
+        <>
+            <CheckCircleOutlined style={{ color: 'var(--success)' }} className="title-icon" />
+            <h2 className="subtitle">You're all set!</h2>
+            <p>
+                You are now subscribed
+                {billing?.is_billing_active && billing.plan && (
+                    <>
+                        {' '}
+                        to the <b>{billing.plan.name}</b>
+                    </>
+                )}{' '}
+                and can use all the premium features immediately.
+            </p>
+            {billing?.plan?.key === 'standard' && (
+                <p className="text-muted-alt">
+                    You will be billed on the <b>first 3 days of each month</b>. If you use less than 1M events, you
```

Comment:
This language is a bit confusing. Should it be "You will be billed in the first 3 days of each month."?